packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [22.0.0]
+
+### Changed
+
+- **BREAKING:** Add peerDependency on `@babel/runtime` ([#3897](https://github.com/MetaMask/core/pull/3897))
+- Remove `cancelMultiplier` & `speedUpMultiplier` option from `TransactionController` ([#3909](https://github.com/MetaMask/core/pull/3909))
+- Throw after publishing a canceled or sped-up transaction if already confirmed ([#3800](https://github.com/MetaMask/core/pull/3800))
+- Bump `eth-method-registry` from `^3.0.0` to `^4.0.0` ([#3897](https://github.com/MetaMask/core/pull/3897))
+
+### Removed
+
+- Remove `cancelMultiplier` and `speedUpMultiplier` constructor options as both values are now fixed at `1.1`. ([#3909](https://github.com/MetaMask/core/pull/3909))
```

Comment:
This looks like it could be a breaking change, since it alters the controller's API that users will interact with?

```suggestion
- **BREAKING:** Remove `cancelMultiplier` and `speedUpMultiplier` constructor options as both values are now fixed at `1.1`. ([#3909](https://github.com/MetaMask/core/pull/3909))
```

Reply:
I thought about that, but I don't think it is. I believe if you're still passing these options then TypeScript won't complain, it'll just invisibly not use them. Is that correct?

Reply:
I'm not sure how that would work. Wouldn't any code that still uses `cancelMultiplier` and `speedUpMultiplier` have to remove those options, since the options object is exhaustively typed? 

In the test file, it might look transaction-controller can silently accept superfluous options, but that's only because `any` type is being used for `...options`, which silences the errors.

Reply:
Oh, right, I was thinking that extraneous properties wouldn't matter due to structural typing, but I forgot [there are special cases where TypeScript does check extraneous properties](https://www.typescriptlang.org/docs/handbook/2/objects.html#excess-property-checks), and this is one of them.

Okay, fair enough. Yeah, it does look like we will want to mark this as breaking:

``` suggestion
- **BREAKING:** Remove `cancelMultiplier` and `speedUpMultiplier` constructor options as both values are now fixed at `1.1`. ([#3909](https://github.com/MetaMask/core/pull/3909))